ipfs-api/src/request/add.rs

```diff
@@ -9,11 +9,43 @@
 use http::Method;
 use request::ApiRequest;
 
-pub struct Add;
+#[derive(Serialize)]
```

Comment:
I think this can derive `Default` to save some typing!

Reply:
Fair point -- I completely forgot that the default for options is none when writing this.